Makefile

```diff
@@ -12,15 +12,17 @@ PACKAGE := github.com/fluid-cloudnative/fluid
 # Go and build settings
 GO_MODULE ?= off
 GC_FLAGS ?= -gcflags="all=-N -l"
-LOCAL_FLAGS ?= -gcflags=-l
+LOCAL_FLAGS ?= -gcflags="all=-N -l"
 CGO_ENABLED ?= 0
-GOOS ?= linux
+GOOS ?= $(shell go env GOOS)
 GOBIN := $(shell if [ -z "$(shell go env GOBIN)" ]; then echo "$(shell go env GOPATH)/bin"; else echo "$(shell go env GOBIN)"; fi)
 
 # Architecture detection
 UNAME := $(shell uname -m)
 ifeq ($(UNAME), aarch64)
```

Comment:
What about handling arm64 identification in just one line?

```bash
ifeq ($(UNAME),$(filter $(UNAME),aarch64 arm64))
    ARCH := arm64
else
    ARCH := amd64
endif
```

Reply:
Done. Thx for the great suggestion.